microcosm_pubsub/tests/test_registry.py

```diff
@@ -104,12 +113,28 @@ def test_find_not_found(self):
 
     def test_register_duplicate(self):
         """
-        No exception raises when re-registering ab existing handler.
+        No exception raises when re-registering an existing handler.
 
         """
         self.registry.register(DerivedSchema.MEDIA_TYPE, noop_handler)
 
-    def test_regiser_another(self):
+    def test_duplicate_bound_handler_found(self):
+        self.graph.unlock()
```

Comment:
Explicitly forcing the graph to have a new handler without (I hope) affecting the other tests.